src/_locales/gr/messages.json

```diff
@@ -0,0 +1,428 @@
+    "description": "The title (tooltip) shown, when hovering over a disabled, managed option."
+  },
+  "optionLearnMore": {
+    "message": "Μάθετε περισσότερα",
+    "description": "When a link to an explainer needs to be added, this is the link text."
+  },
+  "optionsResetButton": {
+    "message": "Επαναφέρετε όλες τις ρυθμίσεις στις προεπιλογές",
+    "description": "The button to delete all current settings and load the defaults, shown in the add-on settings."
+  },
+
+  "titleGeneral": {
+    "message": "Πρόσθετο",
+    "description": "The title for the general settings group."
+  },
+  "titleQrCodeSetting": {
+    "message": "Κωδικός QR",
+    "description": "The title for the qr code settings group."
+  },
+  "titleAddonBehavior": {
+    "message": "Συμπεριφορά πρόσθετων",
+    "description": "The title for the addon behavior settings group."
+  },
+
+  "optionPopupIconColored": {
+    "message": "Έγχρωμο εικονίδιο",
+    "description": "This is an option shown in the add-on settings."
+  },
+  "optionPopupIconColoredDescr": {
+    "message": "Εμφανίζει ένα έγχρωμο εικονίδιο αντί για το ασπρόμαυρο εικονίδιο στη γραμμή εργαλείων.",
+    "description": "This is an option shown in the add-on settings. It describes the optionPopupIconColored setting in more details."
+  },
+
+  "subtitleQrCodeSize": {
+    "message": "Μέγεθος κωδικού QR",
+    "description": "The heading/grouping title shown for the radio buttons for the size setting of the QR code."
+  },
+  "optionQrCodeSizeFixed": {
+    "message": "Σταθερό μέγεθος:",
+    "description": "This is an option shown in the add-on settings. It is one option for selecting the QR code size."
+  },
+  "optionPixelAbbreviation": {
+    "message": "px",
+    "description": "The abbreviation of the term 'pixels'. In most cases, it may stay as it is in English."
+  },
+  "optionQrCodeSizeAuto": {
+    "message": "Αυτόματη προσαρμογή του μεγέθους",
+    "description": "This is an option shown in the add-on settings. It is one option for selecting the QR code size."
+  },
+  "optionQrCodeSizeAutoHelper": {
+    "message": "Κυρίως χρήσιμο σε κινητές συσκευές, όπου ο κωδικός QR ανοίγει σε μια νέα καρτέλα.",
+    "description": "Helper text for the optionQrCodeSizeAuto making the user aware that this setting will likely have no or little effect on desktop devices."
+  },
+  "optionQrCodeSizeRemember": {
+    "message": "Θυμήσου το τελευταίο μέγεθος",
+    "description": "This is an option shown in the add-on settings. It is one option for selecting the QR code size."
+  },
+
+  "subtitleMisc": {
+    "message": "Διάφορες ρυθμίσεις",
+    "description": "The heading/grouping title shown for the miscellaneous setting of the QR code."
+  },
+  "optionQrQuietZone": {
+    "message": "Ήσυχη ζώνη QR:",
+    "description": "This is an option shown in the add-on settings. It's about the area around the QR code."
+  },
+  "optionQrQuietZoneStatusSingular": {
+    "message": "($INTEGER$ square)",
```

Comment:
This seems not to be translated…

Reply:
@GabGaitanidis  If you need any pointers/help or have questions, feel free to ask.